docs/user/06-conditions.md

```diff
@@ -0,0 +1,44 @@
+# Telemetry CR conditions
+
+This section describes the possible states of the Telemetry CR. 
+The state of the Telemetry CR is derived from the combined state of all the subcomponents, namely, from the condition types `LogComponentsHealthy`, `TraceComponentsHealthy` and `MetricComponentsHealthy`. 
+
+## Log Components State
+
+The state of the log components is determined by the status condition of type `LogComponentsHealthy`
+
+| Condition status | Condition reason              | Message                                    |
+|------------------|-------------------------------|--------------------------------------------|
+| True             | ReasonNoPipelineDeployed      | No pipelines have been deployed            |
+| True             | ReasonFluentBitDSReady        | Fluent Bit DaemonSet is ready              |
+| False            | ReasonReferencedSecretMissing | One or more referenced Secrets are missing |
+| False            | ReasonFluentBitDSNotReady     | Fluent Bit DaemonSet is not ready          |
+
+## Trace Components State
+
+The state of the trace components is determined by the status condition of type `TraceComponentsHealthy`
+
+| Condition status | Condition reason                       | Message                                     |
+|------------------|----------------------------------------|--------------------------------------------|
+| True             | ReasonNoPipelineDeployed               | No pipelines have been deployed            |
+| True             | ReasonTraceCollectorDeploymentReady    | Trace collector Deployment is ready        |
+| False            | ReasonReferencedSecretMissing          | One or more referenced Secrets are missing |
+| False            | ReasonTraceCollectorDeploymentNotReady | Trace collector Deployment is not ready    |
+
+## Metric Components State
+
+The state of the metric components is determined by the status condition of type `MetricComponentsHealthy`
+
+| Condition status | Condition reason                         | Message                                     |
+|------------------|------------------------------------------|--------------------------------------------|
+| True             | ReasonNoPipelineDeployed                 | No pipelines have been deployed            |
+| True             | ReasonMetricGatewayDeploymentReady       | Metric gateway Deployment is ready         |
+| False            | ReasonReferencedSecretMissing            | One or more referenced Secrets are missing |
+| False            | ReasonMetricGatewayDeploymentNotReady    | Metric gateway Deployment is not ready     |
+
+
+## Telemetry CR State
+
+The state of Telemetry CR is set to 'Ready' only if all the subcomponent conditions (LogComponentsHealthy, TraceComponentsHealthy, and MetricComponentsHealthy) have a status of 'True.' If any of these conditions are not 'True,' the overall state is set to 'Warning.'
```

Comment:
Playing around with more similar phrasing so it's easier to read quickly :

Option with status at the end (in this case, no change to line 43):

```suggestion
Only if all the subcomponent conditions (LogComponentsHealthy, TraceComponentsHealthy, and MetricComponentsHealthy) have a status of 'True', the state of Telemetry CR is set to 'Ready'.
If any of these conditions are not 'True,' the overall state is set to 'Warning.'
```
Option with status at the beginning (in this case, also adjusting line 43):

```suggestion
- 'Ready': Only if all the subcomponent conditions (LogComponentsHealthy, TraceComponentsHealthy, and MetricComponentsHealthy) have a status of 'True.' 
- 'Warning': If any of these conditions are not 'True'.
- 'Deleting': When a Telemetry CR is being deleted.
- 'Error': If the deletion is blocked because some dependent resources exist.
```